code/game/machinery/firealarm.dm

```diff
@@ -10,7 +10,7 @@
 	result_path = /obj/machinery/firealarm
 
 /obj/machinery/firealarm
-	name = "fire alarm"
+	name = "fire/flood alarm"

```

Comment:
Don't like this, make flood alarms a subtype of fire alarms, so they get their own name, and also give them a blue edge around them, as opposed to the red one of fire alarms.